pages/common/ag.md

```diff
@@ -25,3 +25,7 @@
 - Find files with a name matching "foo":
 
 `ag -g {{foo}}`
+
+- Find files containing "foo" in a specific directory:
+
+`ag {{foo}} {{path/to/folder}}`
```

Comment:
Please make this the 2nd example.

Reply:
Hi @agnivade, my changes were already merged into master. Should I open a new PR for this?

Reply:
Yes. Thank you.